datafusion/common/src/utils.rs

```diff
@@ -162,6 +163,23 @@ where
     Ok(low)
 }
 
+/// This function finds the partition points according to `partition_columns`.
+/// If there are no sort columns, then the result will be a single element
+/// vector containing one partition range spanning all data.
+pub fn evaluate_partition_points(
```

Comment:
Cool, thanks. Some nit: maybe we can invent better naming for `evaluation_partition_points` Partition point I found definitions in Informatica system, not sure if its the same we calculate here. 

Also method returns `lexicographical_partition_ranges` so maybe we can name method similar.

Reply:
I looked up to the Informatica system definition for partition point (I guess what you referred is [this definition](https://docs.informatica.com/data-integration/powercenter/10-5/advanced-workflow-guide/understanding-pipeline-partitioning/partitioning-attributes/partition-points.html)). These are different. If it is misleading, it is better to rename this function. What do you think about renaming it with `evaluate_partition_ranges`?.

Reply:
partition ranges sounds more consistent to me, as in fact in method `if` branches we work to identify exactly ranges.

Reply:
I agree `evaluate_partition_ranges` sounds more specific.

Reply:
I have renamed function `evaluate_partition_points` to `evaluate_partition_ranges`. Thanks for the suggestions @comphead.